test/NuGet.Core.Tests/NuGet.Frameworks.Test/CompatibilityTests.cs

```diff
@@ -599,6 +599,19 @@ public class CompatibilityTests
         [InlineData("netcoreapp1.0", "net451", false)]
         [InlineData("netcoreapp1.0", "net4", false)]
         [InlineData("netcoreapp1.0", "win8", false)]
+
+        // UAP 10.0.15064.0 is compatible with netstandard2.0 
+        // Reference - https://devdiv.visualstudio.com/DevDiv/_queries?id=469873 
+        [InlineData("uap10.0.15064.0", "netstandard2.0", true)]
```

Comment:
Trivial, but you could add the negative test:
`[InlineData("uap10.0.15064.0", "netstandard2.1", false)]`